src/main/java/net/minecraftforge/event/ForgeEventFactory.java

```diff
@@ -253,13 +253,6 @@ public static String getPlayerDisplayName(EntityPlayer player, String username)
         return event.getDisplayname();
     }
 
-    public static List<Predicate<Entity>> gatherEntitySelectors(Map<String, String> map, String mainSelector, ICommandSender sender, Vec3d position)
```

Comment:
To preserve binary compatibility, deprecate this and leave the code in for now.
Leave a javadoc comment saying it should be removed in 1.12